### ISSUE TITLE
Support for tasty-1.4?

### ISSUE THREAD

**Issue by peti** [ocharles/tasty-rerun#20]
The new version of [tasty](https://hackage.haskell.org/package/tasty-1.4) does not compile because of the following errors:

~~~
[   20s] src/Test/Tasty/Ingredients/Rerun.hs:319:27: error:              
[   20s]     • Couldn't match type ‘Tasty.Traversal                      
[   20s]                              (Functor.Compose f0 (Const (Map.Map [Tasty.OptionSet] a0)))’                                                 
[   20s]                      with ‘Tasty.Traversal               
[   20s]                              (Functor.Compose                   
[   20s]                                 (State.StateT Int IO) (Const (Map.Map [String] TestResult)))
[   20s]                            -> Tasty.Traversal                   
[   20s]                                 (Functor.Compose                
[   20s]                                    (State.StateT Int IO) (Const (Map.Map [String] TestResult)))’                                          
[   20s]       Expected type: Tasty.OptionSet                                                                                                      
[   20s]                      -> tasty-1.4:Test.Tasty.Core.TestName                                                                                
[   20s]                      -> Tasty.Traversal                         
[   20s]                           (Functor.Compose                                                                                                
[   20s]                              (State.StateT Int IO) (Const (Map.Map [String] TestResult)))
[   20s]                      -> Tasty.Traversal                                                                                                   
[   20s]                           (Functor.Compose              
[   20s]                              (State.StateT Int IO) (Const (Map.Map [String] TestResult)))                                                 
[   20s]         Actual type: Tasty.OptionSet                         
[   20s]                      -> Tasty.Traversal                                                                          
[   20s]                           (Functor.Compose f0 (Const (Map.Map [Tasty.OptionSet] a0)))                                                     
[   20s]                      -> Tasty.Traversal
[   20s]                           (Functor.Compose f0 (Const (Map.Map [Tasty.OptionSet] a0)))                                                     
[   20s]     • In the ‘foldGroup’ field of a record                      
[   20s]       In the expression:                                        
[   20s]         Tasty.trivialFold                                       
[   20s]           {Tasty.foldSingle = foldSingle, Tasty.foldGroup = foldGroup}                                                                    
[   20s]       In the expression:                                        
[   20s]         let        
[   20s]           foldSingle _ name _ = Tasty.Traversal $ Functor.Compose $ do ...                                                                
[   20s]           foldGroup name children                                                                                                         
[   20s]             = Tasty.Traversal $ Functor.Compose $ do ...                                                                                  
[   20s]         in                                                                                                                                
[   20s]           Tasty.trivialFold                                                                                                               
[   20s]             {Tasty.foldSingle = foldSingle, Tasty.foldGroup = foldGroup}                                                                  
[   20s]     |                                                           
[   20s] 319 |       , Tasty.foldGroup = foldGroup                                                                                                 
[   20s]     |                           ^^^^^^^^^

Would it be possible to get an update for `tasty-rerun` that remedies this issue?

**Comment by peti** [comment]
I think the commit that broke compatibility is https://github.com/feuerbach/tasty/commit/96795203335a54d39e3587c890c2879a3c1a0150.

**Comment by Bodigrim** [comment]
Thanks, will release once Travis builds are green.

**Comment by Bodigrim** [comment]
Released as `tasty-rerun-1.1.18`.
